scalableminds/webknossos
# backend part for versioned volume tracings

- the backend now saves all versions of a volume tracing
- it is possible to revert to a specific version
- documentation changes should be done together with frontend part

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- create a volume tracing
- don't move and create two cells
- send a `revertToVersion` with `"sourceVersion": 1` request to the `data/tracings/volume/:tracingId/update` Route
- reload the tracing
- should show only the first cell

### Issues:
- completes the backend part for #3146 

------
~~- [ ] Updated [changelog](../blob/master/CHANGELOG.md#unreleased)~~
~~- [ ] Updated [migration guide](../blob/master/MIGRATIONS.md#unreleased) if applicable~~
~~- [ ] Updated [documentation](../blob/master/docs) if applicable~~
